mesh/mesh-io-tcpserver.c

```diff
@@ -92,6 +98,11 @@ enum io_type {
 
 static void send_timeout(struct l_timeout *timeout, void *user_data);
 
+static bool tls_ctx_init(void);
+static unsigned int tls_psk_server_cb(SSL * ssl, const char *identity,
+                                      unsigned char *psk,
+                                      unsigned int max_psk_len);
```

Comment:
Don't use forward declarations unless necessary. I think it would be enough to reorder function definitions below.